scripts/destroy_security_group.sh

```diff
@@ -3,17 +3,26 @@
 # Set constant variables
 SECURITY_GROUP_NAME=github-actions-ssh
 
-# Remove the security group from the instance
-aws ec2 modify-instance-attribute --instance-id $AWS_INSTANCE_ID --groups "sg-0c6c1ccf379af096d" "sg-00a5547f58c373231" "sg-0bce9fa27bfbfc695"
-
 # Get the security group name
 group_id=$(aws ec2 describe-security-groups --group-names $SECURITY_GROUP_NAME | jq ".SecurityGroups[0] .GroupId" | tr -d '"')
 
+# Query the instances current security groups
+current_groups=$(aws ec2 describe-security-groups --group-ids $(aws ec2 describe-instances --instance-id $id --query "Reservations[].Instances[].SecurityGroups[].GroupId[]" --output text) --query "SecurityGroups[].GroupId[]")
+
+# Remove all characters we don't want from the original groups
+current_groups=$(echo $current_groups | tr -d '""' | tr -d '[' | tr -d ']' | tr -d ',')
+
+# Remove the group we made from the original groups
+original_groups=$(printf '%s\n' "${current_groups//$group_id/}")
```

Comment:
The double slash here is to do global replacement, and the trailing slash technically has an empty string after it, is that right?

Reply:
Yeah, it uses this synatx: `${string//substring/replacement}` as described here https://www.tldp.org/LDP/abs/html/string-manipulation.html. The double slash means all occurrences